modules/bullet/space_bullet.cpp

```diff
@@ -338,7 +339,7 @@ SpaceBullet::SpaceBullet(bool p_create_soft_world) :
 		gravityMagnitude(10),
 		contactDebugCount(0) {
 
-	create_empty_world(p_create_soft_world);
+	create_empty_world(GLOBAL_DEF("physics/3d/active_soft_world", false));
```

Comment:
Is there any significant performance or memory implications of leaving it on by default?

Reply:
I didn't profiled it, when the soft world is not created all soft solvers are not set.